transformers/test_data/dent.go

```diff
@@ -60,20 +61,18 @@ var DentHeaderSyncLog = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var DentModel = shared.InsertionModel{
+var DentModel = event.InsertionModel{
 	SchemaName: "maker",
 	TableName:  "dent",
-	OrderedColumns: []string{
-		constants.HeaderFK, "bid_id", "lot", "bid", string(constants.AddressFK), constants.LogFK,
+	OrderedColumns: []event.ColumnName{
+		constants.HeaderFK, dent.Id, dent.Lot, dent.Bid, constants.AddressColumn, constants.LogFK,
 	},
-	ColumnValues: shared.ColumnValues{
-		"bid_id":           dentBidId,
-		"lot":              dentLot,
-		"bid":              dentBid,
+	ColumnValues: event.ColumnValues{
+		dent.Id:            dentBidId,
+		dent.Lot:           dentLot,
+		dent.Bid:           dentBid,
 		constants.HeaderFK: DentHeaderSyncLog.HeaderID,
```

Comment:
Not sure when to use one vs the other or if we need both but could we potentially use `event.HeaderFK` from `vulcanizedb/libraries/shared/factories`